packages/hoppscotch-sh-admin/src/pages/users/index.vue

```diff
@@ -109,6 +105,10 @@
               </div>
             </td>
 
+            <td class="py-2 px-7">
+              {{ useTimeAgo(user.lastActiveOn).value }}
```

Comment:
We'll have to account for the case when `lastActiveOn` yields `null`. Currently, it shows `55 years ago` in such a scenario.